orttraining/orttraining/python/training/checkpoint.py

```diff
@@ -3,6 +3,7 @@
 import os
 import torch
 import warnings
+import tempfile
```

Comment:
It seems like ```tempfile``` is not being used. May we remove this import if it is not required?

Reply:
it is used to dump intermediate aggregated checkpoint for Zero+Megatron case